lib/utils/axios-helpers.js

```diff
@@ -0,0 +1,31 @@
+const axios = require('axios');
+
+const auth = require('../auth');
+const config = require('../config');
+
+/**
+ * Set axios' default auth header to OAuth Bearer Token from v5 RO API
+ *
+ * @async
+ * @returns {Promise} If an error is returned from `auth.getToken` the
+ * promise will reject, otherwise it will resolve.
+ *
+ * @example
+ * await setV5Token()
```

Comment:
Let's also add:
```suggestion
 * await setV5Token()
 *
 * @protected
```
given this function is not part of the public API